tests/test_lightning.py

```diff
@@ -103,3 +103,29 @@ def test_lit_training(model, model_kwargs, training_loop):
             max_epochs=2,
         ),
     )
+
+
+def test_lit_pipeline_with_dataset_without_validation():
+    """Test training on a dataset without validation triples."""
+    from pykeen.contrib.lightning import lit_pipeline
```

Comment:
should we try importing this at the top then mark this test as skipped if we fail to import the module? I think we have a utility function for this already

Reply:
https://github.com/pykeen/pykeen/pull/1158/commits/3863d28540ae844f07ca68f3e24c92e45e7b38b7